### PR TITLE
Update schemas

### LINKED ISSUE #4950: Update message schema to include __str__, agent_name, groups, and app_name
Currently the bodhi messaging schema is missing a handful of properties for proper compatibility with upcoming fmn work.

* __str__ -- two messages currently include the __str__ property (which is used to Return a human-readable representation of a message, but longer than the summary i.e. an email body). we need to implement __str__ in the majority of messages:
* agent_name -- we currently already have agent -- just need to expose this as agent_name for full compatability
* app_name -- pretty self explainatory

finally, we also have

* groups -- which i dont think we need to implement in bodhi, as it isnt really aware of FAS groups

### PR DESCRIPTION
**Pull request by ryanlerch**
Resolves #4950

### CODE REVIEW

File: bodhi-messages/bodhi/messages/schemas/update.py [hunk 1]
```diff
@@ -208,6 +220,23 @@ def summary(self) -> str:
...
+
+        Returns:
+            A human readable representation of this message.
+        """
+        new_line = "\n"
+        return (
+            f"{self.update.user.name}'s Bodhi update {self.update.alias}"
+            f"completed push to {self.update.status}\n"
+            f"Builds:\n"
+            f"{new_line.join([b.nvr for b in self.update.builds])} ")
```
Comment (mattiaverga) [inline bodhi-messages/bodhi/messages/schemas/update.py:238]: Is there any limit to a message length? What will happen with updates composed by hundreds of builds?

File: bodhi-messages/bodhi/messages/schemas/update.py [hunk 2]
```diff
@@ -243,6 +272,23 @@ def summary(self) -> str:
...
```
Comment (mattiaverga) [inline bodhi-messages/bodhi/messages/schemas/update.py:290]: Same as above comment.

**Review by mattiaverga — COMMENTED** [review]
Can you drop a change note in the news directory?

File: bodhi-messages/bodhi/messages/schemas/update.py (hunk 1 above)
Comment (ryanlerch) [reply bodhi-messages/bodhi/messages/schemas/update.py:238]: I don't think there is -- @abompard might be able to weigh in on this though.

the spec states that the __str__ is supposed to be like the body of an email (with the summary being the subject) -- so IMHO, its appropriate to list everything here in the __str__

**Comment by ryanlerch** [comment]
> Can you drop a change note in the news directory?

Done!

**Review by mattiaverga — APPROVED** [review]

**Pull request merged**